doc/dev.md

```diff
@@ -213,18 +213,23 @@ xcodebuild -downloadPlatform iOS
 export PATH="$(brew --prefix flex)/bin:$(brew --prefix bison)/bin:$PATH"
 
 # Configure using CMake
+#
+# Make sure to update the `VCPKG_HOST_TRIPLET` parameter below if you are using an x64 MacOS system to the value `x64-osx`. There are
+# also known issues on MacOS, check this issue for the current status: https://github.com/opengisch/QField/issues/5897.
```

Comment:
Can we remove the github issue? I think we want to keep our documentation as time-proof as possible :)

Reply:
Updated it based on the suggestion below, which clarifies it even more.